Sources/ArcGISToolkit/Components/FeatureFormView/FeatureFormView.swift

```diff
@@ -131,6 +146,42 @@ public struct FeatureFormView: View {
 #endif
         .environment(\.validationErrorVisibility, validationErrorVisibility)
         .environmentObject(model)
+        .task {
+            try? await utilityNetwork?.load()
+            if let utilityElement = utilityNetwork?.makeElement(arcGISFeature: model.featureForm.feature) {
+                // Grab Utility Network Associations for the element being edited
+                if let associations = try? await utilityNetwork?.associations(for: utilityElement) {
+                    var groups = [UtilityNetworkAssociationFormElementView.Group]()
+                    // Create a set of the unique network sources present
+                    let networkSources = Array(Set(associations.map { $0.toElement.networkSource }))
+                    for source in networkSources {
+                        // Filter the associations by kind
+                        let groupMembers = associations.filter { $0.toElement.networkSource == source }
+                        var associations: [UtilityNetworkAssociationFormElementView.Association] = []
+                        // For each association, create a Toolkit representation and add it to the group
+                        for association in groupMembers {
+                            let associatedElement = association.toElement
+                            let newAssociation = UtilityNetworkAssociationFormElementView.Association(
+                                description: "\(associatedElement.objectID)",
+                                icon: nil,
+                                name: associatedElement.assetType.name
+                            ) {
+                                if let feature = try? await utilityNetwork?.features(for: [associatedElement]).first {
+                                   return await feature.makeSymbol()
+                                } else {
+                                    return nil
+                                }
+                            }
+                            associations.append(newAssociation)
+                        }
+                        groups.append(.init(associations: associations, description: "[Network Source Description]", name: "\(source.name)".capitalized))
+                    }
+                    self.groups = groups
+                }
```

Comment:
I'm hoping a lot of this logic can be moved into the UN Associations form element.

Reply:
Yep same here, this is just temporary code to get things up and running for now. Planning to pull this out before we merge into v.next.